packages/quantic/cypress/integration/sort/sort-selectors.ts

```diff
@@ -0,0 +1,27 @@
+import {ComponentSelector, CypressSelector} from '../common-selectors';
+
+export const sortComponent = 'c-quantic-sort';
+
+export interface SortSelector extends ComponentSelector {
+  label: () => CypressSelector;
+  combobox: () => CypressSelector;
+  listbox: () => CypressSelector;
+  options: () => CypressSelector;
+  option: (value: string) => CypressSelector;
+  selectedOption: (value: string) => CypressSelector;
+}
+
+export const SortSelectors: SortSelector = {
+  get: () => cy.get(sortComponent),
+
+  label: () => SortSelectors.get().find('.sort__header'),
+  combobox: () => SortSelectors.get().find('.slds-combobox'),
+  listbox: () => SortSelectors.get().find('.slds-listbox'),
+  options: () => SortSelectors.get().find('.slds-listbox__option'),
+  option: (value: string) =>
+    SortSelectors.get().find(`.slds-listbox__option[data-value="${value}"]`),
+  selectedOption: (value: string) =>
```

Comment:
It is kind of weird to pass the expected `value` to this selector. It is like putting the expectation in the selector.

Could we simply have `.slds-listbox__option[aria-checked="true"]` here and then have an expectation that validates the selected element value? If it fails, I think the error message would be more precise.